### DISCUSSION TITLE
Grab function

### DISCUSSION THREAD

**Discussion by mayagomez** [cnr-isti-vclab/TagLab#41] [Ideas]
Hey all, 
I am wondering if it might be possible to introduce a grab function that allows the user to select coral outlines and move them around a plot. 

In hopes to speed up a time series comparison of the same plot, I taglabbed the first time point, exported the shapefile, and then imported that shapefile over the following time point. I am then simply adjusting the coral outlines instead of redoing all of them. However, since the plots are not perfectly aligned, the coral outlines are a bit off as well. I am having to make more drastic edits than expected because of this. A grab function would be really helpful for moving the coral outlines around as needed in plots that are not perfectly aligned and speed up this process. 

Thanks so much, and let me know what you think of this idea and if it might be possible for a future update!
Maya

**Comment by maalechene** [comment]
Hi,
I'm doing the same thing and would be very interested in this function too!
Marine

**Comment by mayagomez** [comment]
Hey Taglab!
Wondering if you happened to see this post and what you think the best way would to TagLab time series plots? 
Thanks!
Maya

**Reply by maxcorsini** [reply]
4:55 PM
Hi, soon we are going to release a co-registration tool integrated in TagLab that should alleviate your problem. Then, you can export the annotation as a label map, import it in the other plot, and validate / adjust the shapes in TagLab directly. Take into account that, for some regions, you do not need to update the shape, but it may be sufficient to refine the boundaries by clicking 'R'. In the next, we are thinking how to transfer the annotation more efficiently, but for the moment, it seems the best way of proceed.

**Reply by mayagomez** [reply]
This is awesome to hear! I am looking forward to the release. Do you have an approx date in mind? Thanks!

**Reply by maxcorsini** [reply]
Hi, around the 23th of September.
